proteobench/utils/quant_datapoint.py

```diff
@@ -0,0 +1,137 @@
+from __future__ import annotations
+
+import dataclasses
+import hashlib
+import logging
+from collections import ChainMap
+from dataclasses import dataclass
+from datetime import datetime
+
+import pandas as pd
+
+
+def filter_df_numquant_median_abs_epsilon(row, min_quant=3):
+    if isinstance(list(row.keys())[0], str):
+        min_quant = str(min_quant)
+    if isinstance(row, dict) and min_quant in row and isinstance(row[min_quant], dict):
+        return row[min_quant].get("median_abs_epsilon")
+    return None
+
+
+def filter_df_numquant_nr_prec(row: pd.Series, min_quant=3):
+    if isinstance(list(row.keys())[0], str):
+        min_quant = str(min_quant)
+    if isinstance(row, dict) and min_quant in row and isinstance(row[min_quant], dict):
+        return row[min_quant].get("nr_prec")
+    return None
+
+
+@dataclass
+class Datapoint:
+    """Data used to stored the"""
+
+    # TODO add threshold value used for presence ion/peptidoform
+    id: str = None
+    software_name: str = None
+    software_version: int = 0
+    search_engine: str = None
+    search_engine_version: int = 0
+    ident_fdr_psm: int = 0
+    ident_fdr_peptide: int = 0
+    ident_fdr_protein: int = 0
+    enable_match_between_runs: bool = False
+    precursor_mass_tolerance: str = None
+    fragment_mass_tolerance: str = None
+    enzyme: str = None
+    allowed_miscleavages: int = 0
+    min_peptide_length: int = 0
+    max_peptide_length: int = 0
+    is_temporary: bool = True
+    intermediate_hash: str = ""
+    results: dict = None
+    median_abs_epsilon: int = 0
+    nr_prec: int = 0
+    comments: str = ""
```

Comment:
Is this the "comments_for_submission"? Or the comments that can manually be added when uploaded the MS file? If it is the latest, should we really keep it? Isn't it confusing to have two comments fields?